app/models/person.rb

```diff
@@ -37,4 +37,17 @@ def roles_inclusion
       errors.add(:roles,"'#{invalid_role}' is a invalid role.")
     end
   end
+  
+  def find_current_institutions
+	current_affiliations = self.affiliations.where("year_to <= #{Date.today.year} OR year_to IS NULL").all
+	institutions = []
+	current_affiliations.each do |current_affiliation|
+	  institutions << current_affiliation.department.institution.name
+	end
+	if institutions.empty?
```

Comment:
Jeśli są puste to nie zwracamy nil-a, bo to nam komplikuje życie. Na końcu powinno być tylko `institutions`